tests/kernel_flexible_centerer.py

```diff
@@ -38,13 +38,15 @@ def test_fit_transform(self):
         """
         K = np.random.uniform(0, 100, size=(3, 3))
         model = KernelFlexibleCenterer()
-        Ktr = model.fit_transform(K)
+        Ktr = model.fit_transform(K.copy())
```

Comment:
Why use `.copy` here?

Reply:
eh, this isn't totally necessary, just a preference when working with modifying functions.

Reply:
Well, why not, ok :).